app/assets/javascripts/modules/track-briefing.js

```diff
@@ -0,0 +1,34 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackBriefing () {}
+
+  TrackBriefing.prototype.start = function($element) {
+    var scope = $element[0]
+
+    this.trackLinks(scope)
+  }
+
+  TrackBriefing.prototype.trackLinks = function(scope) {
```

Comment:
```suggestion

```

Is there a reason to create a separate function for trackLinks here?

Reply:
No reason, just wanted to keep it a bit clean - in case in the future other things need to be added in